jac-uk/admin
# #2294 Use agreed set of v2 statuses

## What's included?

Following review of version 2 Stages & Statuses there are some updates to make to the final set of v2 statuses, as follows:

- [x] Remove new statuses as specified on [spreadsheet](https://docs.google.com/spreadsheets/d/1BjQ6dtrGRWZSQWVPCwB0Q5exiDjUCEIK/edit?usp=sharing&ouid=106025876472425620100&rtpof=true&sd=true)
- [x] Rename display value of new statuses as per column **_v2 Amended Display Value_**
- [x] Relocate Character Rejected status in process according to spreadsheet
- [x] Add new status as per row 23 on spreadsheet
- [x] Remove tasks related to statuses that are going to be removed
- [x] Support statuses which move the application back a stage
- [x] Application list should show processing status rather than application status
- [x] Override tab titles for Diversity and Outreach reports

See this ticket for more detail and a link to the spreadsheet: jac-uk/digital-platform#1002

See [Candidate process flow](https://miro.com/app/board/uXjVN7KsLv8=/)

Closes #2294 

## Who should test?
✅ Product owner
✅ Developers

## How to test?
Example exercise: [2294 Stages and statuses](https://jac-admin-develop--pr2325-feature-2294-l1fi4273.web.app/exercise/EzikIl5VGsMblAcCCCu1/stages/applied), [Stages and statuses test1](https://jac-admin-develop--pr2325-feature-2294-l1fi4273.web.app/exercise/Oy4hPcHarUc0CMQau7Hh/dashboard)

- Navigate to Stages
- For each of the stages select an application then click the Set status button
- On the Update page check that the correct statuses are displayed

(refer to spreadsheet mentioned above)

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context

**Applied**
<img width="684" alt="image" src="https://github.com/jac-uk/admin/assets/8524401/61969019-855b-4d47-b1df-1463b5615dec">


**Shortlisted**
<img width="691" alt="image" src="https://github.com/jac-uk/admin/assets/8524401/9ad89ad2-c918-450a-b847-8e070cd197ee">


**Selectable**
<img width="667" alt="image" src="https://github.com/jac-uk/admin/assets/8524401/4ef86817-6386-4dd1-96c9-cbc7c48fef98">


**Recommended**
<img width="682" alt="image" src="https://github.com/jac-uk/admin/assets/8524401/c7859b10-6b4d-4eb8-a44c-bb0ba82173f7">






## Related permissions
Have permissions been considered for this functionality?
- No permission changes required
- Permissions have been added / updated. Details:

---
PREVIEW:PRODUCTION
_can be OFF, DEVELOP or STAGING_
